vms/job/services.py

```diff
@@ -81,6 +80,31 @@ def get_jobs_by_event_id(e_id):
     return job_list
 
 
+def get_jobs_by_date(start_date, end_date):
```

Comment:
get this change in enhancement commit and not in test one

Reply:
Done